enclave/src/channel_storage.rs

```diff
@@ -51,14 +53,16 @@ pub enum ChannelType {
 
 pub fn load_sender() -> Result<Sender<ChannelType>, ChannelStorageError> {
```

Comment:
I know it's a hassle, but could you write unit tests for this function? I.e. what happens when you load three senders, does it still work or will there be a deadlock? Does the created receiver get all functions call?

Or is it not possible to write unit tests for this?

Reply:
We do have a problem with tests here, they conflict with the already running Receiver and `substratee-worker test --unit` never finishes running, after running the tests it gets stuck in the receiver loop because it's sender was replaced by the one in the test.

Reply:
`load_sender` should not replace anything, should it?

Reply:
In order to test load_sender i also need a receiver to check the message being sent is also being received, and for that i need to create a new channel, leaving the DB Thread receiver without a sender.

Reply:
I see, then a follow up question: Why are you initializing the DBThread for testing purposes here?
https://github.com/Polkadex-Substrate/polkadexTEE-worker/pull/240/files#diff-8bf3b6ae639112751d4db557e3fb729745967cc35382ee32c386488580ffb6c3R36

Reply:
Because otherwise the tests that use balance will fail because changes in the balance storage trigger a send to the receiver.

Reply:
https://github.com/Polkadex-Substrate/polkadexTEE-worker/issues/246